index.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
-var Raven = require('raven')
+const raven = require('raven')
 
-exports.register = function (server, options, next) {
-  Raven.config(options.dsn, options.client)
-  server.expose('client', Raven)
-  server.on('request-error', function (request, err) {
-    var baseUrl = request.info.uri ||
-      request.info.host && `${server.info.protocol}://${request.info.host}` ||
+exports.register = (server, options) => {
```

Comment:
no arrow functions unless it's one line and/or you need this binding

Reply:
Because I use standard and it's a difficult rule to lint for. As I said, you're welcome to use arrow functions when it's one line or to bind this.

Reply:
ever thought about using a linter with more simple rules? eg. https://eslint.org/

Reply:
I do use eslint. I use standard, it uses eslint. It has the right balance of rules and personal effort (none). I'm not interested in maintaining every style preference I have in a linter across 400+ OSS projects.

Reply:
It sounds like you're expecting to send a PR and either the linter tells you the style is wrong or it's 100% ok. That's not how I handle projects. I make a lot of style decisions loosely for fairly specific reasons. Standard is mostly focused on a few minor areas of consistency and carving out a syntax that's safe with ASI.

Reply:
Yes I do, because that's the only sense of linting...
Do your style decisions once, put them into a central eslint-config repo and use this eslint-config in all your other repos:
```
  "devDependencies": {
    "@bendrucker/eslint-config": "^1.2.3"
  },
  "eslintConfig": {
    "extends": "@bendrucker/eslint-config/base"
  }
```